docs/persistent-storage.md

```diff
@@ -0,0 +1,120 @@
+
+Here is an example from an actual MVP:
+```
+pvc:
+  volName: social-platform-static-data
+  pvcName: social-platform-pvc
+  mountPath: "/app/data"
+  labels:
+    billingType: "monthly"
+    region: us-south
+    zone: dal10
+  spec:
+    accessModes:
+      - ReadWriteMany
+    resources:
+      requests:
+        storage: 20Gi
+    storageClassName: ibmc-file-bronze    
+```
+
+## 2. Create the PVC template
+
+Next, create a PVC template called `pvc.yaml` in the `chart/app-name/templates` folder.  This will define your PVC resource making use of the values provided in the `values.yaml` file. The `pvc.yaml` would be like the following:
+```
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: "{{  .Values.pvc.pvcName  }}"
+  labels:
+    billingType: "{{  .Values.pvc.labels.billingType  }}"
+    region: "{{  .Values.pvc.labels.region  }}"
+    zone: "{{  .Values.pvc.labels.zone  }}"
+spec:
+  accessModes: {{  .Values.pvc.spec.accessModes  }}
+  resources:
+    requests:
+      storage: "{{  .Values.pvc.spec.resources.requests.storage  }}"
+  storageClassName: "{{  .Values.pvc.spec.storageClassName  }}"
+```
+
+## 3. Update `deployment.yaml`
+
+Next, mount the volume to your deployment, by updating `deployment.yaml` in `chart/app-name/templates` folder.  Under the `containers` section, where the container is defined, add a `volumeMounts` section as shown below:
+```
+    spec:
+      containers:
+      - name: "{{  .Chart.Name  }}"
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        volumeMounts:
+        - name: "{{  .Values.pvc.volName  }}"
+          mountPath: "{{  .Values.pvc.mountPath  }}"
+```
+
+Further at the bottom, after your containers are defined, add the `volumes` definition at the same indentation level as `containers`:
+```
+      volumes:
+      - name: "{{  .Values.pvc.volName  }}"
+        persistentVolumeClaim:
+          claimName: "{{  .Values.pvc.pvcName  }}"
+```
+
+This defines your PV and PVC for deployment of the application.  
+
+## 4. Verify PVC and File Storage
+
+Once you have deployed the application to Kubernetes using the Helm charts with the PVC addition, verify that the PVC and PV resources have been created and the persistent file storage path persists even after the pod has been removed.
+
+To verify whether the PVC is created, use the following command:
+```
+kubectl describe pvc <pvc name>
+```
+
+To verify that the PV is successfully mounted:
+```
+kubectl describe deployment <deployment_name>
+```
+
+You can verify the file storage, by updaing the data in your `<mount path>`.  Next, delete the pod with the application, which will trigger Kubernetes to create a new pod with the application.  If the persistent volume for the path is operational, then you should still see the updated data in that `<mount path>` as compared to being reset without the persistent volume.
```

Comment:
Change:  `"Next, delete the pod with the application, which will trigger Kubernetes to create a new pod with the application."`    ---> To --->  `"Next, delete the application's pod (using either the Kubernetes console or CLI), this will trigger Kubernetes to create a new pod for this application."`